packages/environment-check/src/checks/service-checks.ts

```diff
@@ -246,6 +246,10 @@ export async function checkTransportRequests(
         }
     } catch (e) {
         logger.error(t('error.getTransportRequestsError'));
+
+        if (e.message.includes('403')) {
```

Comment:
I would make this more robust
```
if (e.message?.includes('403')) {
```

Reply:
Thanks @longieirl , ai have made a change.